client/src/app/containers/Alert/slice.js

```diff
@@ -0,0 +1,27 @@
+import { nanoid } from '@reduxjs/toolkit';
+import { createSlice } from '@reduxjs/toolkit';
+
+export const initialState = {
+  alerts: [],
+};
+
+const alertSlice = createSlice({
+  name: 'alert',
+  initialState,
+  reducers: {
+    addAlert: {
+      reducer: (state, action) => {
+        state.alerts = [...state.alerts, action.payload];
```

Comment:
:cactus: could use `push` or `unshift` instead, don't know if it matters.

Reply:
I've seen `push` elsewhere 🤷